R/detectStandard.R

```diff
@@ -33,11 +33,19 @@ detectStandard <- function(data, domain="labs"){
   standard_list[["details"]][["SDTM"]]<-compare_cols(data_cols,getRequiredColumns(standard="SDTM"))
   
   # Determine the final standard
-  if(standard_list[["details"]][["SDTM"]][["match"]]){
+  if(standard_list[["details"]][["SDTM"]][["match"]] == "Full"){
```

Comment:
Could also consider going with `full_match` and `partial_match` booleans ... although this is probably fine.

Reply:
yeah i did that originally then went with one setting - gonna keep it in mind as we go forward